### ISSUE TITLE
aria-haspopup should be able to have the popups role as a value

### ISSUE THREAD

**Issue by w-green** [evcohen/eslint-plugin-jsx-a11y#360]
According to the docs aria-haspopup should be able to have a boolean or the role of the associated popup container.

> Authors must ensure that the role of the element that serves as the container for the popup content is menu, listbox, tree, grid, or dialog, and that the value of aria-haspopup matches the role of the popup container

http://w3c.github.io/aria/aria/aria.html#aria-haspopup

Currently it flags as an error if one of the roles is used.
`error  The value for aria-haspopup must be a boolean`
